### DISCUSSION TITLE
How to specify seed for `BatchExecutor`?

### DISCUSSION THREAD

**Discussion by lostmsu** [SciSharp/LLamaSharp#991] [General]
By default it seems to repeatably produce the same sequence.

**Comment by martindevans** [comment]
the seed is something you set on the sampler, rather than on the executor.

So in [this example](https://github.com/SciSharp/LLamaSharp/blob/master/LLama.Examples/Examples/BatchedExecutorSaveAndLoad.cs#L38) you would modify the `sampler` object like so:

```csharp
var sampler = new DefaultSamplingPipeline()
{
    Seed = 1234,
};
```
